libs/designsystem/src/lib/scss/base/_typography.scss

```
@@ -28,6 +28,38 @@ kirby-app .page-title {
   }
 }
 
+ul,
+.ul,
+ol,
+.ol {
+  list-style: none; /* Remove default bullets/numbers */
+  padding-left: 8px;
+  counter-reset: item;
+}
+
+ul,
+ol {
+  li {
+    display: flex;
+    line-height: 1.2em;
+  }
+}
+
+ul li::before {
```

We should probably use the child combinator (`>`) here to avoid font-size getting larger and larger for each nested level of lists.

```suggestion
ul > li::before {
```